charts/ndb-operator/templates/validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: ndb-operator-system/ndb-operator-serving-cert
```

Comment:
I feel this needs to be templated too.

Reply:
I am tending towards holding off templating the webhook-related internal stuff (as long as that is not causing issues with helm install). ?

So I will hold addressing it off, until I test installing the operator with a different name and see if it works.